torch/_subclasses/fake_tensor.py

```diff
@@ -320,9 +327,14 @@ def mk_fake_tensor(make_meta_t):
         )
         if out is NotImplemented:
             raise UnsupportedFakeTensorException("meta converter nyi")
+        if isinstance(t, torch._subclasses.functional_tensor.FunctionalTensor):
+            # breakpoint()
+            out = torch._subclasses.functional_tensor.FunctionalTensor(torch._to_functional_tensor(out))
```

Comment:
instead of hardcoding the handling here, I think it might make more sense to handle it where we handle fakeifying all of the other custom tensor types:

(1) in `describe_tensor()`, add a piece of state saying if our tensor is a `FunctionalTensor`: https://github.com/pytorch/pytorch/blob/main/torch/_subclasses/meta_utils.py#L203

(2) add handling in `def meta_tensor()` that branches on that state, and recursively handles `t.elem`

Reply:
Currently made the `FunctionalTensor` case recurse back into `from_real_tensor` since `MetaConverter` doesn't handle `FakeTensor`s and the pattern we need to handle is `FunctionalTensor(_to_functional_tensor(FakeTensor(...)))`.